src/Data/SongDifficulty.cpp

```diff
@@ -22,12 +23,21 @@ namespace SongDetailsCache {
     }
 
     bool SongDifficulty::ranked() const noexcept {
-        return stars > 0 && song().rankedStatus == RankedStatus::Ranked;
+        return this->rankedBL() || this->rankedSS();
```

Comment:
minor, but please put rankedSS() in front, I think that'd be better for this as SS is more likely to have a song ranked atm